modules/service-cc.sh

```diff
@@ -0,0 +1,62 @@
+# shellcheck disable=SC2034,SC2039
+ 
+CC_PROVISIONING_SERVICE_TITLE="Canonical Common Criteria EAL2 Provisioning"
+CC_PROVISIONING_SUPPORTED_SERIES="xenial"
+CC_PROVISIONING_SUPPORTED_ARCHS="x86_64 ppc64le s390x"
+
+CC_PROVISIONING_REPO_URL="https://private-ppa.launchpad.net/ubuntu-advantage/commoncriteria"
+CC_PROVISIONING_REPO_KEY_FILE="ubuntu-cc-keyring.gpg"
+CC_PROVISIONING_REPO_LIST=${CC_PROVISIONING_REPO_LIST:-"/etc/apt/sources.list.d/ubuntu-cc-${SERIES}.list"}
+CC_PROVISIONING_UBUNTU_COMMONCRITERIA="ubuntu-commoncriteria"
+
+cc_provisioning_enable() {
+    local token="$1"
+    local result=0
+
+    _cc_is_installed || result=$?
+    if [ $result -eq 0 ]; then
+        error_msg "Common Criteria artifacts are already installed and available in /usr/lib/common-criteria."
+        error_exit service_already_enabled
+    fi
+
+    check_token "$CC_PROVISIONING_REPO_URL" "$token"
+    apt_add_repo "$CC_PROVISIONING_REPO_LIST" "$CC_PROVISIONING_REPO_URL" "$token" \
+                 "${KEYRINGS_DIR}/${CC_PROVISIONING_REPO_KEY_FILE}"
+    apt_install_package_if_missing_file "$APT_METHOD_HTTPS" apt-transport-https
+    apt_install_package_if_missing_file "$CA_CERTIFICATES" ca-certificates
+    echo -n 'Running apt-get update... '
+    check_result apt_get update
+    echo 'Ubuntu Common Criteria PPA repository enabled.'
+
+    echo -n 'Installing Common Criteria artifacts (this may take a while)... '
+    # shellcheck disable=SC2086
+    check_result apt_get install $CC_PROVISIONING_UBUNTU_COMMONCRITERIA
+
+    echo -n "Successfully prepared this machine to host the Common Criteria artifacts. "
+    echo "Please follow instructions in /usr/lib/common-criteria/README to configure EAL2 on the target machine(s)."
+}
+
+cc_provisioning_disable() {
+    not_supported 'Disabling Common Criteria EAL2 Provisioning'
+}
```

Comment:
Since this is now just installing and removing a single deb, we can probably think about allowing disabling. It's a topic for another branch, though.